web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +83,21 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  it("render cards with a zero count in the For You header when no Fundings and no certifications are present", () => {
```

Comment:
[boulder] I feel like there is a missing test(s) for any number other than zero... we're not actually testing the functionality that it counts up certifications and fundings.

Reply:
added a test for certification and funding.